app/views/guide/part_payments.html.slim

```diff
@@ -44,26 +44,26 @@ header
     .panel
       h3 id="what-to-do" What to do
       
-      h5 When correct payment is received:
+      h5 When correct part-payment is received:
 
       ul
-        li select the application from the list ‘Awaiting payment’
-        li log the payment as being received
+        li select the application from the list ‘Waiting for part-payment' 
```

Comment:
The heading on the dashboard currently says 'Waiting for payment' not 'Waiting for part-payment